packages/schemas/tables/hooks.sql

```diff
@@ -1,13 +1,16 @@
 create table hooks (
-    tenant_id varchar(21) not null
-      references tenants (id) on update cascade on delete cascade,
-    id varchar(21) not null,
-    event varchar(128) /* @use HookEvent */ not null,
-    config jsonb /* @use HookConfig */ not null,
-    created_at timestamptz not null default(now()),
-    primary key (id)
-  );
+  tenant_id varchar(21) not null
+    references tenants (id) on update cascade on delete cascade,
+  id varchar(21) not null,
+  name varchar(256),
+  event varchar(128) /* @use HookEvent */, /* deprecated */
```

Comment:
will the deprecated mark shows in the generated typescript type?

Reply:
No, it won't.

Reply:
should we mark it as well?

Reply:
Good idea; I will support it in another issue. LOG-6091

Reply:
```suggestion
  event varchar(128) /* @use HookEvent; @deprecated */,
```

Reply:
This `@deprecated` mark will be supported in LOG-6091, and removed temporarily.